client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-upgrade-plan/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { PLAN_BUSINESS, getPlan } from '@automattic/calypso-products';
+import { StepContainer } from '@automattic/onboarding';
+import { UpgradePlan } from 'calypso/blocks/importer/wordpress/upgrade-plan';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { useSite } from 'calypso/landing/stepper/hooks/use-site';
+import { useSiteSlug } from 'calypso/landing/stepper/hooks/use-site-slug';
+import { goToCheckout } from 'calypso/landing/stepper/utils/checkout';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const SiteMigrationUpgradePlan: Step = function ( { navigation } ) {
+	const siteItem = useSite();
+	const siteSlug = useSiteSlug();
```

Comment:
I had to use this because `siteItem.slug` was null, 🤔

Reply:
Are you testing with the siteSlug= param on the URL?

Reply:
yup, `siteItem.ID` works 🙀